docs/metric/README.md

```diff
@@ -0,0 +1,47 @@
+
+## Minio Prometheus Metric
+
+Minio server exposes an endpoint for Promethueus to scrape server data at `/minio/metric`.
+
+### Prometheus  probe
+
+Prometheus is used to monitor Minio server information like http request, disk storage, network stats etc.. It uses a config file named `prometheus.yaml` to scrape data from server.
+The value for `metrics_path` and `targets` need to be configured in the config yaml to specify the endpoint and url as shown:
+
+
+```
+scrape_configs:
+  # The job name is added as a label `job=<job_name>` to any timeseries scraped from this config.
+  - job_name: minio
+    metrics_path: /minio/metrics
+
+    # metrics_path defaults to '/metrics'
+    # scheme defaults to 'http'.
+
+    static_configs:
+      - targets: ['localhost:9000']
+```
+
+ Prometheus can be run by executing :
+```
+
```

Comment:
unnecessary space after code block begin

Reply:
Removed the unnecessary space